tests/pr_test.go

```diff
@@ -216,3 +219,60 @@ func TestRunUpgradeVpcPattern(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestRunOverride(t *testing.T) {
+	t.Parallel()
+
+	options := setupOptionsQuickStartPattern(t, "slz-ovr", quickStartPatternTerraformDir)
+	options.SkipTestTearDown = true
+	output, err := options.RunTestConsistency()
+
+	if assert.Nil(t, err, "This should not have errored") &&
+		assert.NotNil(t, output, "Expected some output") &&
+		assert.NotNil(t, options.LastTestTerraformOutputs, "Expected some Terraform outputs") {
+		// set override json string with previous value of config output
+		options.TerraformOptions.Vars["override_json_string"] = options.LastTestTerraformOutputs["config"]
+
+		options.SkipTestSetup = true
+		// TERRATEST uses its own internal logger.
+		// The "show" command will produce a very large JSON to stdout which is printed by the logger.
+		// We are temporarily turning the terratest logger OFF (discard) while running "show" to prevent large JSON stdout.
+		options.TerraformOptions.Logger = logger.Discard
+		planStruct, err := terraform.InitAndPlanAndShowWithStructE(options.Testing, options.TerraformOptions)
```

Comment:
More of a golang style recommendation, I have seen issues in the past when reusing variables on the left hand side of the `:=` operator, in this case reusing the variable `err` that was used for previous function call. To avoid any runtime issues you could make sure that you use unique error variables for each call (like for instance call this one `errPlan` or similar). Sometimes golang deals with it ok, and other times it complains, I've been taking the safe route and using unique names.